### PR TITLE
Bumps k8s-kvm image

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/3593

### CODE REVIEW

File: service/controller/v13/version_bundle.go [hunk 1]
```diff
@@ -22,6 +22,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Added CA public key into trusted user keys for SSO ssh.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "k8s-kvm",
```
Comment (JosephSalisbury) [inline service/controller/v13/version_bundle.go]: Is this the right name? Discuss plz.

**Comment by JosephSalisbury** [comment]
For my understanding - I'm adding the fix into the latest version, and then we'll update the clusters to here to fix (as opposed to backport weirdnesses).

File: service/controller/v13/version_bundle.go (hunk 1 above)
Comment (MarcelMue) [reply service/controller/v13/version_bundle.go]: Something like `k8s-kvm-docker-image` would be more precise and fairly obvious to me if I was a customer.
But we did name the repo just `k8s-kvm` so I am unsure here as well.

**Review by MarcelMue — APPROVED** [review]
LGTM

**Comment by JosephSalisbury** [comment]
```
$ gsctl create cluster --owner=giantswarm --name=joe-entropy-test-2 --release=2.10.0
Requesting new cluster for organization 'giantswarm'
New cluster 'joe-entropy-test-2' (ID '3j3gs') for organization 'giantswarm' is launching.
Add key pair and settings to kubectl using

    gsctl create kubeconfig --cluster=3j3gs

Take into consideration all guest clusters have enabled RBAC and may you want to provide a correct organization for the certificates (like operators, testers, developer, ...)

    gsctl create kubeconfig --cluster=3j3gs --certificate-organizations system:masters

To know more about how to create the kubeconfig run

    gsctl create kubeconfig --help

```
```
$ kubectl get nodes
NAME                            STATUS    ROLES     AGE       VERSION
master-kd93u-dd85c774b-ggcds    Ready     master    12m       v1.10.4
worker-ws0ja-58db9c4c4b-h458z   Ready     <none>    12m       v1.10.4
```
```
$ kubectl describe node worker-ws0ja-58db9c4c4b-h458z | grep IP
  InternalIP:  172.23.1.158
```
```
$ opsctl ssh geckon master1
Warning: Permanently added 'master1.geckon.gridscale.kvm.gigantic.io' (ECDSA) to the list of known hosts.
Last login: Wed Jul  4 09:42:29 UTC 2018 from 10.0.4.254 on pts/0
Container Linux by CoreOS stable (1576.5.0)
Update Strategy: No Reboots
giantswarm@master-10-0-4-101 ~ $ ssh 172.23.1.158
The authenticity of host '172.23.1.158 (172.23.1.158)' can't be established.
ECDSA key fingerprint is SHA256:CdzNnqanaUnLLsBrbPlBaIAKVNx2g6aQXn0ChlPUxcU.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added '172.23.1.158' (ECDSA) to the list of known hosts.
Container Linux by CoreOS stable (1688.5.3)
Update Strategy: No Reboots
giantswarm@worker-ws0ja-58db9c4c4b-h458z ~ $ ls /dev/hwrng
/dev/hwrng
```

File: service/controller/v13/version_bundle.go (hunk 1 above)
Comment (JosephSalisbury) [reply service/controller/v13/version_bundle.go]: I have zero opinion on this. If anyone else from @giantswarm/sig-operator has one, I'd be happy to hear it.

**Pull request merged**